src/scripts/modules/countdown-timer.js

```diff
@@ -0,0 +1,60 @@
+const init = () => {
+  Array.from(document.querySelectorAll('.countdownTimer'))
+    .forEach((timer) => {
+      let days = 0;
+      let hours = 0;
+      let minutes = 0;
+      let seconds = 0;
+      let count = 0;
+
+      const daysElement = timer.querySelector('.countdownTimer-days span');
+      const hoursElement = timer.querySelector('.countdownTimer-hours span');
+      const minutesElement = timer.querySelector('.countdownTimer-mins span');
+      const secondsElement = timer.querySelector('.countdownTimer-secs span');
+      const endDate = new Date(timer.dataset.date).getTime();
+
+      if (typeof (endDate) !== 'number') {
+        return;
+      }
+
+      const calculate = () => {
+        count += 1;
+
+        let startDate = new Date();
+        startDate = startDate.getTime();
+
+        // turn milliseconds into seconds to make calculations easier
+        let timeRemaining = parseInt((endDate - startDate) / 1000, 10);
+
+        if (timeRemaining >= 0) {
```

Comment:
Whilst there's nothing wrong with this, it could potentially be turned into an early-return.